.github/workflows/logger-mutants.yml

```diff
@@ -0,0 +1,30 @@
+name: Logging Mutants
+
+# only run on push in order to update the cache output
+# flow:
+# restore cache
+# install cargo-mutants crate in order to run the 'cargo mutants' command
+# create a file with the current commit hash if a previous one doesn't exist, then print it
+# run the script that handles the 'cargo mutants' command on the differences between the latest updates and the last commit where it was ran
+# overwrite the previous commit hash with the current one for the following run
+# delete the old cache
+# save the new cache with the updated mutants
+# upload artifact to easily check it for the given commit
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - next
+
+jobs:
+  save_cache:
+    runs-on: ubuntu-latest
+
+    # test
+    steps:
+      - name: Run logging mutants from actions
+        uses: stacks-network/actions/mutation-testing/logger@feat/mutation-testing
```

Comment:
this workflow doesn't seem to exist on this branch (or in main). is that something that needs to be re-added?

Reply:
I'd recommend merging the `feat/mutation-testing` branch into stacks-network/actions first, so you don't have to make another PR changing/removing the branch name later

Reply:
Also, I can't find the `logger` action either

Reply:
> Also, I can't find the `logger` action either

i expect this PR will be changing quite a bit as we've just started testing this. i'm converting this to draft for now

Reply:
yes, i will be updating it these days, wanted to have the `PR-differences` fully finished asap. this one will have to wait till we have all the mutants from the general sync before being merged